### PR TITLE
pull request : changes from db-changes to master

### PR DESCRIPTION
**Pull request by ShanGemunu**

### CODE REVIEW

File: Queries.php [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+require_once __DIR__ . '/vendor/autoload.php';
+
+use Dotenv\Dotenv;
+
+$dotenv = Dotenv::createImmutable(__DIR__);
+$dotenv->load();
+
+class Queries
+{
+    function getMsrnRanges($conn)
```
Comment (pasindu-dil) [inline Queries.php]: Use access modifiers for all functions.
  private function getMsrnRanges($conn)

File: Queries.php [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+        $statement = $conn->prepare($query);
+        if (!($statement->execute())) {
+            throw new Exception('exception occured in insertIntoMsrn');
+        }
+        $statement->close();
+    }
+
+
+    function alterTableCdrCall($conn){
+        $query = "ALTER TABLE `cdr_call_test_01` ADD `roaming` TINYINT NULL DEFAULT '0' AFTER `network_id`";
```
Comment (pasindu-dil) [inline Queries.php]: Change table name as "cdr_call".

File: Queries.php [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+
+    
+
+    function alterTableSystemParamOne($conn){
+        $query = "
+        ALTER TABLE `system_parameters`
+        ADD PRIMARY KEY (`id`),
+        ADD KEY `param` (`param`)
+        ";
+
```
Comment (pasindu-dil) [inline Queries.php]: No need alter table query for system_parameters. Add keys when creating a table.

File: Queries.php [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+        $statement = $conn->prepare($query);
+        if (!($statement->execute())) {
+            throw new Exception('exception occured in alterTableSystemParamOne');
+        }
+    }
+    
+
+    function alterTableSystemParmTwo($conn){
+        $query = "ALTER TABLE `system_parameters` CHANGE `id` `id` INT(11) NOT NULL AUTO_INCREMENT";
```
Comment (pasindu-dil) [inline Queries.php]: You can add auto_increment when creating a table.

File: Queries.php [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+        $statement = $conn->prepare($query);
+        if (!($statement->execute())) {
+            throw new Exception('exception occured in alterTableCdrCall');
+        }
+    }
+
+    
+    function createTableSystemParameters($conn){
+        $query = "
+        CREATE TABLE `system_parameters` (
```
Comment (pasindu-dil) [inline Queries.php]: Could you check that the table exists before creating a table?

CREATE TABLE IF NOT EXISTS `system_parameters` ...

File: Queries.php [hunk 6]
```diff
@@ -0,0 +1,287 @@
...
+
+        $statement = $conn->prepare($query);
+        if (!($statement->execute())) {
+            throw new Exception('exception occured @insertIntoSystemParamOne');
+        }
+    }
+
+    function insertIntoSystemParamTwo($conn)
+    {
+        $count = $count = $this->getCountForSystemParamByColumn('param', 'rotation_delay_on_traffic_trends', $conn);
```
Comment (pasindu-dil) [inline Queries.php]: No need double assignment in hear!

File: index.php [hunk 7]
```diff
@@ -1,2 +1,45 @@
 <?php
+require_once __DIR__ . '/Connecter.php';
+require_once __DIR__ . '/Queries.php';
+
+$connecter = Connecter::getConneterInstance();
```
Comment (DasunMuthuruwan) [inline index.php]: create connection inside Queries.php file

File: Queries.php (hunk 1 above)
Comment (ShanGemunu) [reply Queries.php]: done

File: Queries.php (hunk 2 above)
Comment (ShanGemunu) [reply Queries.php]: done

File: Queries.php (hunk 3 above)
Comment (ShanGemunu) [reply Queries.php]: done

File: Queries.php (hunk 4 above)
Comment (ShanGemunu) [reply Queries.php]: done

File: Queries.php (hunk 5 above)
Comment (ShanGemunu) [reply Queries.php]: done

**Pull request merged**